client/src/App.tsx

```diff
@@ -6,6 +6,7 @@ import { Home } from './modules/Home'
 import { Course } from './modules/Course'
 import { Profile } from './modules/Profile'
 import { Admin } from './modules/Admin'
+import { AdminCosting } from './modules/AdminCosting'
```

Comment:
One thing I'm not quite sure about, but could be a more clean way to separate the admin page and its functionality from the rest of the app - maybe Admin could contain a router of its own so that new pages to Admin are not all added to the overall app.